src/api/helpers.jl

```diff
@@ -411,6 +419,11 @@ function h5p_get_efile_prefix(plist)
     return String(buffer)
 end
 
+function h5p_set_efile_prefix(plist, sym::Symbol)
+    sym === :origin ? h5p_set_efile_prefix(plist, raw"$ORIGIN") :
+    error("The only valid symbol for h5p_set_efile_prefix is :origin.")
```

Comment:
Is this what you meant?

```suggestion
    throw(ArgumentError("The only valid `Symbol` argument for `h5p_set_efile_prefix` is `:origin`. Got `$sym`."))
```

Reply:
Yes exactly, perhaps you missed my comment above?

Reply:
I saw your comment. It has an odd number of backticks.

Reply:
Ah, typo.